### PR TITLE
# IDEMPIERE-5488 - adding missing ids in .tdp file and regenerating .…

### PR DESCRIPTION
**Pull request by matheus-marcelinux**
…target

https://idempiere.atlassian.net/browse/IDEMPIERE-5488

# Pull Request Checklist

## Checklist:

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have added unit tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
-target "idempiere-220825"
+target "idempiere-221125"
 
 with source configurePhase allEnvironments
 
-location "https://idempiere.github.io/binary.file/p2.zk/9.6.2" {
+location binary-file-zk "https://idempiere.github.io/binary.file/p2.zk/9.6.2" {
```
Comment (hengsin) [inline org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd:6]: please simplify to zk

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd [hunk 2]
```diff
@@ -17,7 +17,7 @@ location "https://idempiere.github.io/binary.file/p2.zk/9.6.2" {
 	zweb-dsp
 }
 
-location "https://idempiere.github.io/binary.file/p2.repackaged/9.0.0" {
+location binary-file-repackaged "https://idempiere.github.io/binary.file/p2.repackaged/9.0.0" {
```
Comment (hengsin) [inline org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd:20]: please simplify to repackaged

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd [hunk 3]
```diff
@@ -27,7 +27,7 @@ location "https://idempiere.github.io/binary.file/p2.repackaged/9.0.0" {
 	org.apache.poi.ooxml-schemas	
 }
 
-location "https://download.eclipse.org/eclipse/updates/4.20" {
+location eclipse-2021-06 "https://download.eclipse.org/eclipse/updates/4.20" {
```
Comment (hengsin) [inline org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd:30]: please simplify to eclipse

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd [hunk 4]
```diff
@@ -145,7 +145,7 @@ location orbit-2021-06-R20210602031627 "https://download.eclipse.org/tools/orbit
 	org.apache.commons.discovery	
 }	
 
-location "https://sourceforge.net/projects/jasperstudio/files/updatesite/6.17.0" {
+location jasperstudio-updatesite "https://sourceforge.net/projects/jasperstudio/files/updatesite/6.17.0" {
```
Comment (hengsin) [inline org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd:148]: please simplify to jasperstudio

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd (hunk 3 above)
Comment (matheus-marcelinux) [reply org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd]: done

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd (hunk 2 above)
Comment (matheus-marcelinux) [reply org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd]: done

File: org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd (hunk 1 above)
Comment (matheus-marcelinux) [reply org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.tpd]: done

**Review by hengsin — APPROVED** [review]
trivial fix, good to go

**Pull request merged**